src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -78,7 +78,7 @@
           + "'biller', biller, "
           + "'billingAccountId', billing_account_id, "
           + "'description', description, "
-          + "'cloudPlatform', cloud_platform, "
+          + "'cloudPlatform', lower(cloud_platform), "
```

Comment:
Why is this needed now when it worked before this PR?

Reply:
I checked and it didn't :( try running locally with a breakpoint and getting a summary...the cloud platform will always be null